### ISSUE TITLE
Allow the app to be installed as a PWA

### ISSUE THREAD

**Issue by JeDaYoshi** [gitpod-io/openvscode-server#120]
The app already includes a `manifest.json` -  but it's lacking some details. Most notably:
- It has no Service Worker included, so Chrome etc. says "no"
- the app lacks any proper icons (although this also applies to favicons) - maybe some should be added :)

**Comment by bigint** [comment]
+1 love to see this, code-server has this great option, but I love Gitpod 😄

**Comment by tlockney** [comment]
It does allow you to at least create the app on iPadOS in Safari, but it uses the redirect URL without the token, so when you try to open it, you get a "Forbidden" message.

**Comment by filiptronicek** [comment]
Please see https://github.com/microsoft/vscode/issues/135553 for tracking progress on this issue.

**Comment by hanneshayashi** [comment]
Seems like this has been implemented now upstream in https://github.com/microsoft/vscode/issues/135553.

**Comment by filiptronicek** [comment]
> Seems like this has been implemented now upstream in microsoft#135553.

Yes indeed, thanks for noticing. Before we close this, I want to wait until we ship insiders today, I have a feeling this will not work right away since this mainly affects [vscode.dev](https://vscode.dev/).

**Comment by jrderuiter** [comment]
Any updates on this? (Would love to have this feature to address keyboard shortcuts etc.)

**Comment by Nishchit14** [comment]
This is a very needy feature (also easy to implement), I don't know why it's taking a year-long time. I am coming from a codespace and the experience is not matching due to PWA here in the GitPod.

**Comment by bbigras** [comment]
Is this still relevant?
